### DISCUSSION TITLE
Can not filter multiselect?

### DISCUSSION THREAD

**Discussion by thomaskamalakis** [Budibase/budibase#6367] [Help] [answered]
Hi everyone

  I do not seem to be able to add filter to a Provider based on multiselect columns. So if I have a budibaseDB table `thesis` containing just a `test` multiselect column taking values `A` and `P` and a title text field, I can not build a table view containing just the rows containing `P`

  An export of the app can be found [here](https://drive.google.com/file/d/1J37brd0v_ae_zKM6y8BPnWG-IqrPSrz4/view?usp=sharing)

  Has anyone seen this?

  BR

  Thomas

**Comment by melohagan** [comment, marked as answer]
Hey @thomaskamalakis 

I think this open issue is related: https://github.com/Budibase/budibase/issues/3773

**Reply by melohagan** [reply]
This functionality should be available soon in the next release: https://github.com/Budibase/budibase/pull/6815
